src/mixins/filing-template-mixin.ts

```diff
@@ -239,6 +241,15 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
       }
     }
 
+    // If a Transactional Folio Number was entered then override the folio number
+    // in the filing and save a flag to correctly restore a draft if needed.
+    const fn = this.getFolioNumber
+    const tfn = this.getTransactionalFolioNumber
+    if (tfn !== fn) {
```

Comment:
I did it this way because '' is false and we wanted the ability to save an empty FN.

So now, if the TFN is different from the business FN, just use it. This means there is always a FN in the filing header (unless they're all empty of course).